packages/netlify-cms/webpack.config.js

```diff
@@ -1,32 +1,66 @@
 const path = require('path');
-const webpack = require('webpack');
 const FriendlyErrorsWebpackPlugin = require('friendly-errors-webpack-plugin');
 const CopyWebpackPlugin = require('copy-webpack-plugin');
-const pkg = require('./package.json');
-const { plugins } = require('../../scripts/webpack');
-const coreWebpackConfig = require('../netlify-cms-core/webpack.config.js');
+const pkg = require(path.join(process.cwd(), 'package.json'));
 
 const isProduction = process.env.NODE_ENV === 'production';
 
 const baseConfig = {
-  ...coreWebpackConfig,
-  context: path.join(__dirname, 'src'),
-  entry: './index.js',
+  mode: isProduction ? 'production' : 'development',
+  context: path.join(__dirname, 'dist'),
+  entry: './netlify-cms.esm.js',
```

Comment:
Is this really correct? I _think_ webpack needs the raw source file. This could explain the large UMD bundle as well